docs/changelog.md

```diff
@@ -1,5 +1,64 @@
 ## Changelog
 
+### v2.17.0 (28/9/2021)
+
+#### Highlights
+
+- Support [PVT](https://arxiv.org/abs/2102.12122) and [PVTv2](https://arxiv.org/abs/2106.13797)
+- Support large scale jittering and New Mask R-CNN baselines
+- Speedup `YOLOv3` inference
+- Add a general data structrue for the results of models
```

Comment:
line 10-11 can be removed from highlights and we need to update the `support of SOLO`.